airflow/models/dag.py

```diff
@@ -92,11 +91,34 @@
 
 log = logging.getLogger(__name__)
 
-ScheduleInterval = Union[str, timedelta, relativedelta]
 DEFAULT_VIEW_PRESETS = ['tree', 'graph', 'duration', 'gantt', 'landing_times']
 ORIENTATION_PRESETS = ['LR', 'TB', 'RL', 'BT']
 
+ScheduleIntervalArgNotSet = type("ScheduleIntervalArgNotSet", (), {})
```

Comment:
Is there a reason this isn't just `object()`?

Does this display better somewhere?

Reply:
Only reason is for typing; `func(arg=object())` is impossible to type because any `Union` with `object` ends up being just `object`.